src/renderers/server-render.jsx

```diff
@@ -190,19 +182,15 @@ const makeRenderer = (
 	})
 	.first(state => state.preRenderChecklist.every(isReady => isReady));  // take the first ready state
 
-	console.log(JSON.stringify({
-		message: `Dispatching RENDER for ${request.url.href}`,
-		type: 'dispatch',
-		info: {
-			url: request.url,
-			method: request.method,
-			id: request.id,
-		}
-	}));
-	store.dispatch({
+	const action = {
 		type: SERVER_RENDER,
 		payload: url,
-	});
+	};
+	server.app.logger.debug(
```

Comment:
yeah, this has a similar constraint to importing the `config` instance in different parts of the application, and the resulting setup is similar. We should always prefer to read from `server.app.logger` because it is by definition the server-side logging instance and there's no need for an extra import when you have access to `server`.

The need to import `logger` directly might be an indication that a module could/should be refactored to be better integrated into the `server`.

I'll add some documentation, but in practice it won't matter if people import `logger` directly, but using `server.app.logger` is cleaner